src/Sulu/Bundle/AdminBundle/Tests/Functional/Metadata/FormMetadata/FormMetadataProviderTest.php

```diff
@@ -15,9 +15,9 @@
 use Sulu\Bundle\AdminBundle\Metadata\FormMetadata\FormMetadata;
 use Sulu\Bundle\AdminBundle\Metadata\FormMetadata\FormMetadataProvider;
 use Sulu\Bundle\AdminBundle\Metadata\FormMetadata\TypedFormMetadata;
-use Sulu\Bundle\TestBundle\Testing\KernelTestCase;
+use Sulu\Bundle\TestBundle\Testing\SuluTestCase;
 
-class FormMetadataProviderTest extends KernelTestCase
+class FormMetadataProviderTest extends SuluTestCase
```

Comment:
Why do we need the `SuluTestCase` here? Shouldn't the simpler `KernelTestCase` be sufficient? What is the difference between these two in general?

Reply:
you are right KernelTestCase is here enough